Source/Initialization/WarpXAMReXInit.H

```diff
@@ -31,7 +31,7 @@ namespace warpx::initialization
         int& argc,
         char**& argv,
         bool const build_parm_parse = true,
```

Comment:
Interesting - this one should not be const either.
I thought we have a separate clang tidy check for that which catches this...